packages/react-native/src/primitives/Radio/types.ts

```diff
@@ -0,0 +1,37 @@
+import { PressableProps, StyleProp, TextStyle, ViewStyle } from 'react-native';
+
+import { LabelPosition } from '../Label/types';
+
+export type Size = number | 'small' | 'medium' | 'large';
```

Comment:
This type could be shared among other components.

Reply:
I agree. Is there a place we can put shared types?

Reply:
Let's move it in to the Icon types once that PR has been merged

Reply:
But will Icon `size` accept `'small' | 'medium' | 'large'`?